ext/hal/cmsis/Kconfig

```diff
@@ -1,8 +1,10 @@
-#
 # Copyright (c) 2016 Intel Corporation
-#
 # SPDX-License-Identifier: Apache-2.0
-#
 
-config HAS_CMSIS
+config HAS_CMSIS_CORE
```

Comment:
So instead of:

```
config HAS_CMSIS_CORE_M
	bool
	depends on HAS_CMSIS_CORE
```

Let's have:

```
if HAS_CMSIS_CORE

config HAS_CMSIS_CORE_M
	bool

endif
```

Reply:
Done